src/main/cpp/t03_toggle.cpp

```diff
@@ -24,5 +24,16 @@
 using namespace std;
 
 int t03_toggle() {
-
+    char s1,s2;
+    int diff=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'diff' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4487945)